virtualizarr/xarray.py

```diff
@@ -127,7 +148,9 @@ def open_virtual_dataset(
                 filepath=filepath, reader_options=reader_options
             )
 
-            ds = xr.open_dataset(fpath, drop_variables=drop_variables)
+            ds = xr.open_dataset(
+                fpath, drop_variables=drop_variables, decode_times=False
```

Comment:
This is a change that required changes to tests (see: cebb031) but I _do_ think this is more correct. We don't want to be decoding things if it isn't explicitly requested right?

Reply:
@TomNicholas really want to draw attention to this

Reply:
I agree. We can also make this more auto-magic later if we want. I'm not sure any of the existing tests were covering this.